src/LiquidityPool.sol

```diff
@@ -192,8 +189,8 @@ contract LiquidityPool is Auth, IERC7540 {
     }
 
     /// @inheritdoc IERC4626
-    function maxDeposit(address operator) external view returns (uint256 maxAssets) {
-        maxAssets = manager.maxDeposit(address(this), operator);
+    function maxDeposit(address owner) external view returns (uint256 maxAssets) {
```

Comment:
I think `owner` is very confusing here, as it is not necessarily the initial owner that locked the tokens. It can also be a different receiver address... I would go with a more neutral naming like `user` and only distinguish between owner and receiver if required by the context.
Same applies to all the other `max` functions

Reply:
It is the owner of the request, that is the idea of the latest iteration of the 7540 spec. That when you submit a request, you specify a receiver, which "receives the request" and is then the owner of the request going forward. I think this makes sense. Also because the param names of these max* methods in the 4626 spec are owner.

Reply:
I agree we can possibly change this in the investment manager implementation, but in liquidity pool I don't think we should deviate from the 7540 & 4626 specs.